metadata-ingestion/src/datahub/ingestion/source/nifi.py

```diff
@@ -638,7 +638,12 @@ def create_nifi_flow(self):
         )
         nifi_version: Optional[str] = None
         if about_response.ok:
-            nifi_version = about_response.json().get("about", {}).get("version")
+            try:
+                nifi_version = about_response.json().get("about", {}).get("version")
+            except Exception as e:
```

Comment:
if response already has 200 Ok status then its unclear why this would raise an exception. If it still raises exception here then this looks okay.